code/API_definitions/location-verification.yaml

```diff
@@ -297,18 +336,18 @@ components:
           - `PARTIAL`: when the requested area partially match the area where the network locates the device. A `match_rate` could be included in the response.
       type: string
       enum:
-        - 'TRUE'
-        - 'FALSE'
-        - 'UNKNOWN'
-        - 'PARTIAL'
+        - "TRUE"
+        - "FALSE"
+        - "UNKNOWN"
+        - "PARTIAL"
     MatchRate:
       description: Match rate estimation for the location verification in percent
```

Comment:
Estimation of the match rate between Requested Area R and Network Area N, calculated as the percent value of Area(R ∩ N) / Area(N). Included only if VerificationResult is PARTIAL.

Reply:
Yes agree -- probably better to be very explicit. 
description: Match rate estimation for the location verification in percent - _may be valued only for PARTIAL result_.

Reply:
Rephrased